app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
```

Comment:
For the Registration, I suggest using Devise gem (https://github.com/heartcombo/devise), it handles all Sign In and Sign up stuff 😃

Reply:
As I discuss with @hoangmirs, I decided to implement everything from scratch to understand how Ruby on Rails work. In this case, do I still need to use `Devise`?

Reply:
Oh I see, next time you could mention that in the PR description so the other reviewer know about that and won't make the comment like this 🙌

Reply:
But I recommend taking a look on Devise (later) as on the real project we only use Devise :)

Reply:
Got it Andy, thanks